Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -180,9 +180,10 @@ def source_buffer(node)
         node.source_range.source_buffer
       end
 
-      # Returns the string representation if node is of type str
+      # Returns the string representation if node is of type str(plain) or dstr(interpolated)
       def string_content(node)
-        node.str_content if node.type == :str
+        return node.str_content if node.type == :str
+        node.each_child_node(:str).map(&:str_content).join("") if node.type == :dstr
```

Comment:
There is execute-string https://github.com/whitequark/parser/blob/master/doc/AST_FORMAT.md#execute-string
I'm not sure if we should consider that here. Can you please take a look and confirm that?